lib/tether/interfaces.go

```diff
@@ -37,6 +37,7 @@ type Operations interface {
 	Apply(endpoint *NetworkEndpoint) error
 	MountLabel(ctx context.Context, label, target string) error
 	MountTarget(ctx context.Context, source url.URL, target string, mountOptions string) error
+	CopyExistingContent(source string) error
```

Comment:
> Is there a reason we do this after all of the volumes have been mounted? Another option would have been to inline this behaviour with the MountTarget logic.

I chose to follow this approach to avoid duplicating the code in MountLabel and MountTarget functions. 

If I understood your second question correctly,
Contents are not copied from VolumeA to VolumeB, they are always from the base image to a volume. One, cannot mount two volumes on the same path.